api/src/game.ts

```diff
@@ -49,10 +59,8 @@ export const switchToNextPlayer = (
   actualPlayer: Player,
   players: Array<Player>,
 ): Player => {
-  let nextPlayerIndex = players.indexOf(actualPlayer) + 1;
-
-  if (nextPlayerIndex > players.length - 1) return players[0];
-  return players[nextPlayerIndex];
+  if (actualPlayer.playerNumber === 1) return players[1];
+  return players[0];
 };
 
 export const marbleWonByPlayer = (graph: Graph): number => {
```

Comment:
I don't understand what this function is supposed to do and return. Does it make a marble be won by a player or does it check whether a marble has been won by a player?

Reply:
This function iterate thought the graph to check marbles in the outside nodes.
And return the value of the node found outside if there is one.
changed to `marbleColorExited`

Reply:
`marbleColorExited` does seems clearer to me XD